src/tests/smokeTest.spec.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * POC test for automated UI tests for Planner
+ *  Story: Display and Update Work Item Details
+ *  https://github.com/almighty/almighty-core/issues/298
+ *
+ * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
+ * Tests will be run on these resolutions:
+ * - iPhone6s - 375x667
+ * - iPad air - 768x1024
+ * - Desktop -  1440x900
+ *
+ * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly.
+ *
+ * @author naina-verma, rgarg@redhat.com
+ */
+
+import { browser, element, by, ExpectedConditions, ElementFinder, protractor } from 'protractor';
+
+import { TestSupport } from './testSupport';
+import { Constants } from './constants';
+import { WorkItemListPage } from './pages/work-item-list.page';
+
+var testSupport = new TestSupport();
+var constants = new Constants();
+
+describe('Work item list', function () {
+  var page, items, browserMode;
+
+  var until = protractor.ExpectedConditions;
+  var WORK_ITEM_TITLE = "The test workitem title";
+  var WORK_ITEM_UPDATED_TITLE = "Test workitem title-UPDATED";
+  var WORK_ITEM_DESCRIPTION = "The test workitem description";
+  var WORK_ITEM_UPDATED_DESCRIPTION = "Test description-UPDATED";
+  var EXAMPLE_USER_0 = "Example User 0";
+  var EXAMPLE_USER_1 = "Example User 1";
+  var MOCK_WORKITEM_TITLE_0 = "Title Text 0";
+  var WORKITEM_0_ID = 'id0';
+  var WORKITEM_1_ID = 'id1';
+  var AREA_0_TITLE = '/Root Area/Area 0';
+  var AREA_1_TITLE = '/Root Area/Area 1';
+  var newLabelTitle = "My Test Label";
+  var testLabelTitle = "Example Label 0";
+
+  beforeEach(function () {
+    testSupport.setBrowserMode('desktop');
+    page = new WorkItemListPage(true);
+    testSupport.setTestSpace(page);
+  });
+
+  /* User can read, update, remove assignee on a workitem  */
+  it('User can read, update, remove assignee', function() {
+    page.typeQuickAddWorkItemTitle(WORK_ITEM_TITLE);
+    page.clickQuickAddSave().then(function() {
```

Comment:
These tests are still the exact same as JS tests. Renaming JS file to TS file doesn't actually fix the issues with the tests

Reply:
Yes, the test ports are only there for testing the test runner. They will be replaced with your new tests. This PR is only about the runner, not the tests.